docs/index.asciidoc

```diff
@@ -299,10 +298,8 @@ If you'd like to use an HTTP proxy . This supports multiple configuration syntax
   * Value type is <<number,number>>
   * Default value is `60`
 
-This module makes it easy to add a very fully configured HTTP client to logstash
-based on [Manticore](https://github.com/cheald/manticore).
-For an example of its usage see https://github.com/logstash-plugins/logstash-input-http_poller
-Timeout (in seconds) for the entire request
+This module helps you add a fully configured HTTP client to Logstash based on https://github.com/cheald/manticore[Manticore].
```

Comment:
This section looks like it has been copy/pasted from [here](https://github.com/logstash-plugins/logstash-mixin-http_client/blob/master/lib/logstash/plugin_mixins/http_client.rb#L4), and seems a bit out of place here in the middle of the configuration settings.

It can probably be replaced by 

```
Timeout (in seconds) for the entire request
```

It would be nice to have configuration examples as in the [http poller input docs](https://www.elastic.co/guide/en/logstash/current/plugins-inputs-http_poller.html#_example)